config.toml

```diff
@@ -21,3 +21,13 @@ internal_rules_organizations_csv_file = ""
 
 [metrics]
 namespace = "smart_proxy"
+
+[sentry]
+dns = "sentry logging dns"
+
+[kafka_zerolog]
```

Comment:
Wouldn't it be better to leave this empty, or something like that? Just to avoid unexpected surprised with the logger in case it is not resilient to kafka connexion errors and such